packages/taquito-michelson-encoder/src/errors.ts

```diff
@@ -3,7 +3,7 @@ export { MapValidationError } from './tokens/map';
 export { SetValidationError } from './tokens/set';
 export { BigMapValidationError } from './tokens/bigmap';
 export { TokenArgumentValidationError, TokenComparisonError } from './tokens/pair';
-export { OrTokenDecodingError } from './tokens/or';
+export { OrTokenDecodingError, OrValidationError as OrValidationError } from './tokens/or';
```

Comment:
curious the reason of casting as itself here?

Reply:
I think this has happened during an automatic rename. I removed it. Thanks